tests/Resources/Page/OCPDashboard/Page.robot

```diff
@@ -11,4 +11,12 @@ Open Page
 
 Page Should Be Open
   [Arguments]  ${url}
-  Location Should Contain  ${url}
\ No newline at end of file
+  Location Should Contain  ${url}
+
+Select Project By Name
+  [Arguments]  ${project_name}
+  Wait Until Page Contains Element    xpath://div/button[contains(@class, 'co-namespace-dropdown__menu-toggle')]
+  # Select From List By Value  xpath://*/section/h1[.="Projects"]/ul  ${project_name}
```

Comment:
can you remove the commented code here and few places below?

Reply:
yes, will do!